cluster/sync.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+#
+# Copyright 2018-2022 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+source ./cluster/cluster.sh
+cluster::install
+
+if [[ "$KUBEVIRT_PROVIDER" == external ]]; then
```

Comment:
Seems not relevant to our repo.

Reply:
why not ? we can allow external, it is a kubevirtci flag basically
(even that the name is misleading)
it will set the registry and such accordingly